app/[id]/events/new/setup-event-users.tsx

```
@@ -1,76 +1,68 @@
-            <View className="flex m-5 gap-2">
-                <View className="flex flex-row justify-end " >
-                    <Button
-                        onPress={() => trip?.users.filter(user => !containsUser(user, attendees)).forEach((u) => append(u))}>
-                        <Text className="underline text-blue-400 font-bold">Sélectionner tous</Text>
-                    </Button>
+                <View className="m-5 gap-2">
+                    <Text className="font-bold text-2xl dark:text-white">
+                        Ajoute des participants
+                    </Text>
+                    <Text className="text-gray-400">
+                        Ajoute tout de suite tes amis qui participeront à l'activité. Tu n'es pas obligé de faire ça dès maintenant, tes amis
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)